File: front/src/models/Attendee.ts
import type {EventDate, EventDateState} from "@/models/EventDate.ts";

export class Attendee {

  private name: string;

  public constructor(name: string) {
    this.name = name;
  }

  public getName() : string {
    return this.name;
  }
}

export interface AttendeeState {
  name : string
}

File: back/src/main/java/be/naaturel/letsmeet/core/models/Attendee.java
package be.naaturel.letsmeet.core.models;

import java.util.Objects;

public class Attendee {

    private String name;

    public Attendee(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) return true;
        if (obj == null || getClass() != obj.getClass()) return false;
        Attendee date = (Attendee) obj;
        return Objects.equals(name, date.name);
    }

    @Override
    public int hashCode() {
        return Objects.hash(name);
    }
}
